src/main/java/seedu/address/model/person/staff/Shift.java

```diff
@@ -0,0 +1,136 @@
+package seedu.address.model.person.staff;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.AppUtil.checkArgument;
+
+import java.time.DayOfWeek;
+import java.time.LocalTime;
+import java.time.format.DateTimeParseException;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+
+
+/**
+ * Represents a single shift of a staff member in the restaurant book.
+ */
+public class Shift implements Comparable<Shift> {
```

Comment:
Given that the day of the week is cyclic, does a (linear) comparator really make sense?

Reply:
The comparator here is for sorting shifts, to allow shifts to be printed out in a logical order. There's otherwise no need to define some kind of partial order for shifts.